test/copy.tilelive.test.js

```diff
@@ -93,6 +93,35 @@ test('copy mbtiles', function(t) {
   });
 });
 
+test('copy mbtiles with v1 tile logging', function(t) {
+  process.env.LOG_V1_TILES = true; 
+  var fixture = path.resolve(__dirname, 'fixtures', 'valid.mbtiles');
+  var src = 'mbtiles://' + fixture;
+  var dst = dsturi('valid.mbtiles');
+  sinon.spy(tilelive, 'copy');
+
+  tileliveCopy(src, dst, {}, function(err) {
+    t.ifError(err, 'copied');
+    tileCount(dst, function(err, count) {
+      t.ifError(err, 'counted tiles');
+      t.equal(count, 21, 'expected number of tiles');
```

Comment:
since we're just testing for logging, let's get rid of these expected count checks - since they are flaky anyways.